internal/testutil/acc/cluster.go

```diff
@@ -6,12 +6,14 @@ import (
 	"testing"
 
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/constant"
+	"go.mongodb.org/atlas-sdk/v20240530002/admin"
 )
 
 type ClusterRequest struct {
-	ProviderName           string
-	ExtraConfig            string
 	ResourceDependencyName string
+	ClusterNameExplicit    string
+	ReplicationSpecs       []admin.ReplicationSpec
```

Comment:
was expecting `ClusterRequest` to contain `ReplicationSpecRequest`, so when making the call to the function you dont have to worry about converting into the SDK model and have knowledge of `acc.ReplicationSpec`.

Reply:
Great point! Changed in 8f84795afa65b708cac743f0fa6f6ea644c7aae5